src/main/java/address/Browser/TabBrowser.java

```diff
@@ -0,0 +1,60 @@
+package address.browser;
+
+import com.teamdev.jxbrowser.chromium.javafx.BrowserView;
+import javafx.scene.control.Tab;
+import javafx.scene.control.TabPane;
+
+import java.util.ArrayList;
+
+/**
+ * A TabPane UI that consists of a browser with multiple tabs.
```

Comment:
Shouldn't put implementation detail (`TabPane UI`) in comment because the implementation can be changed although the class still does the same thing. `A browser with multiple tabs` suffices.